package.json

```diff
@@ -49,7 +54,8 @@
     "sequelize": "^5.15.0",
     "sequelize-cli": "^5.5.0",
     "swagger-ui-express": "^4.0.7",
-    "underscore": "^1.9.1"
+    "underscore": "^1.9.1",
+    "validatorjs": "^3.15.1"
   },
   "devDependencies": {
     "@babel/cli": "^7.5.5",
```

Comment:
Why are the `src/__mocks__/emailVerification`, `src/__tests__/emailverification.js` and  `src/__tests__/unknownRoute.js` files deleted?